### PR TITLE
backblast particles

### PR DESCRIPTION
**Pull request by tyeagg**
## About The Pull Request
backblast particles for rocket launchers

https://user-images.githubusercontent.com/98577688/224560364-205ea63e-25f7-4d88-a34b-da9a75227912.mp4
## Why It's Good For The Game
looks nice
## Changelog
:cl:
add: particles for rl backblast
/:cl:

### CODE REVIEW

**Comment by novaepee** [comment]
Does Alamo also release backblast particles when flying away?

**Review by Lumipharon — APPROVED** [review]
Neat

File: code/modules/projectiles/guns/specialist.dm [hunk 1]
```diff
@@ -556,26 +572,23 @@ Note that this means that snipers will have a slowdown of 3, due to the scope
...
 	var/turf/blast_source = get_turf(src)
 	var/thrown_dir = REVERSE_DIR(get_dir(blast_source, target))
 	var/turf/backblast_loc = get_step(blast_source, thrown_dir)
-	smoke.set_up(0, backblast_loc)
-	smoke.start()
+	var/angle = Get_Angle(loc, target)
+	var/x_component = sin(angle) * -30
+	var/y_component = cos(angle) * -30
+	backblast = new(blast_source, /particles/backblast)
+	backblast.particles.velocity = list(x_component * 1, y_component * 1)
```
Comment (TiviPlus) [inline code/modules/projectiles/guns/specialist.dm:589]: Multiplying by 1?

File: code/modules/projectiles/guns/specialist.dm [hunk 2]
```diff
@@ -556,26 +572,23 @@ Note that this means that snipers will have a slowdown of 3, due to the scope
...
+	QDEL_IN(backblast, 0.6 SECONDS)
+
```
Comment (TiviPlus) [inline code/modules/projectiles/guns/specialist.dm:591]: Should use QDEL_NULL_IN or even better just add the particle holder to vis contents as needed so you dont need to recreate it every time

**Review by TiviPlus — CHANGES_REQUESTED** [review]

**Pull request merged**